js/javascript-single-thread-event-loop-work-flow

```diff
@@ -0,0 +1,43 @@
+In this Blog, we'll learn about How JavaScript actually works under the hood.
+
+We have always heard people mentioning that **JavaScript is Single Threaded.**
+### What Does that means?
+In very simple language it means that, JS is running at most one line of JS code at a time, it can switch back and forth between different lines of code very very quickly so we don't really think that only one thing is happening at a time.
```

Comment:
since we are referring to language as programming languages, it would be confusing to say language in any other meaning. Let's change it to "simple terms"

very or very very is not needed.